### PR TITLE
Introduce card add-label command

### PR DESCRIPTION
**Pull request by diepsohung**

### CODE REVIEW

File: lib/3llo/utils.rb [hunk 1]
```diff
@@ -67,5 +67,11 @@ def build_req_path(path, extra_params = {})
 
       [path, URI.encode_www_form(params)].join("?")
     end
+
+    def require_folder_files(current_file, suffix)
```
Comment (qcam) [inline lib/3llo/utils.rb:71]: Let's move this change in another PR.

Comment (diepsohung) [reply lib/3llo/utils.rb:71]: Okay man.

File: lib/3llo/command/card/add_label.rb [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+            add_label(card_id, label_ids)
+            interface.puts("Chosen labels have been added to the card.")
+          end
+        end
+
+        private
+
+        def assert_card_id!(card_id, key)
+          raise InvalidArgumentError.new("#{key.inspect} is not a valid list key") unless card_id
```
Comment (qcam) [inline lib/3llo/command/card/add_label.rb:26]:
```suggestion
          raise InvalidArgumentError.new("#{key.inspect} is not a valid card key") unless card_id
```

File: lib/3llo/command/card/add_label.rb [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        end
+
+        def add_label(card_id, label_ids)
```
Comment (qcam) [inline lib/3llo/command/card/add_label.rb:29]:
```suggestion
        def add_labels(card_id, label_ids)
```

File: lib/3llo/command/card/add_label.rb [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+          label_ids =
+            card.labels.flat_map do |label|
+              index = label_options.find_index { |_label_name, label_id| label_id == label.id }
+
+              index ? [index + 1] : []
+            end
+
+          interface.input.multi_select(
+            "Choose the label to add this card to:",
```
Comment (qcam) [inline lib/3llo/command/card/add_label.rb:47]:
```suggestion
            "Choose the labels to add to this card:",
```

File: lib/3llo/command/card/add_label.rb [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+          API::Card.add_labels(card_id, label_ids)
+        end
+
+        def select_label(interface, card, board_id)
+          label_options =
+            API::Label.find_all_by_board(board_id)
+              .map { |label| [label.name, label.id] }
+              .to_h()
+
+          label_ids =
```
Comment (qcam) [inline lib/3llo/command/card/add_label.rb:39]:
```suggestion
          preselected_label_ids =
```

File: lib/3llo/command/card/add_label.rb [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+
+        def assert_card_id!(card_id, key)
+          raise InvalidArgumentError.new("#{key.inspect} is not a valid card key") unless card_id
+        end
+
+        def add_labels(card_id, label_ids)
+          API::Card.add_labels(card_id, label_ids)
+        end
+
+        def select_label(interface, card, board_id)
```
Comment (qcam) [inline lib/3llo/command/card/add_label.rb:33]:
```suggestion
        def select_labels(interface, card, board_id)
```

File: lib/3llo/command/card/add_label.rb [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+        def execute(key, board_id)
+          card_id = Entities.parse_id(:card, key)
+          assert_card_id!(card_id, key)
+
+          card = API::Card.find(card_id)
+
+          interface = Application.fetch_interface!()
+
+          interface.print_frame do
+            label_ids = select_label(interface, card, board_id)
```
Comment (qcam) [inline lib/3llo/command/card/add_label.rb:16]:
```suggestion
            label_ids = select_labels(interface, card, board_id)
```
